pkg/getter/plugingetter_test.go

```diff
@@ -144,3 +144,27 @@ func TestGetterPlugin(t *testing.T) {
 
 	assert.Equal(t, "fake-plugin output", buf.String())
 }
+
+func TestCollectGetterPluginsPassesEnv(t *testing.T) {
+	env := cli.New()
+	env.PluginsDirectory = pluginDir
+	env.Debug = true
+
+	providers, err := collectGetterPlugins(env)
+	require.NoError(t, err)
+	require.NotEmpty(t, providers, "expected at least one plugin provider")
+
+	getter, err := providers.ByScheme("test")
+	require.NoError(t, err)
+
+	gp, ok := getter.(*getterPlugin)
+	require.True(t, ok, "expected getter to be a *getterPlugin")
+
+	require.NotEmpty(t, gp.env, "expected env to be set on getterPlugin")
+	envMap := plugin.ParseEnv(gp.env)
```

Comment:
ParseEnv is exposed just for this test. It is an internal package though, so not as big a deal.

Reply:
If you want me to duplicate the logic instead, I can. I thought it would not be an issue to publish it as it is an internal package.